app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryimage/EntryImageActivity.java

```diff
@@ -186,8 +214,24 @@ public void showContestStatusScreen() {
         startActivity(ContestStatusActivity.makeIntent(this));
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.M)
+    @Override
+    public void checkStoragePermissions() {
```

Comment:
The external storage permission is also declared in the manifest so we shouldn't need an alternative here for pre-M devices